Vagrantfile

```diff
@@ -0,0 +1,39 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+VAGRANTFILE_API_VERSION = '2'
+
+Vagrant.require_version '>= 1.5.0'
+
+Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
+  config.vm.hostname = 'instance-image'
+  if Vagrant.has_plugin?('vagrant-omnibus')
+    config.omnibus.chef_version = 'latest'
+  end
+  if Vagrant.has_plugin?('vagrant-berkshelf')
+    config.berkshelf.enabled = true
+  end
+  config.vm.box = 'chef/centos-6.6'
+  config.vm.network :private_network, type: 'dhcp'
+  config.vm.provision :chef_solo do |chef|
+    chef.json = {
+      ganeti: {
+        :"master-node" => true,
```

Comment:
Mixing hash styles is a bad idea. I recommend you convert everything to the hash rocket syntax.